client/my-sites/email/email-management/page.jsx

```diff
@@ -0,0 +1,18 @@
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+import EmailManagementHome from 'calypso/my-sites/email/email-management/email-home';
+import { emailManagement } from 'calypso/my-sites/email/paths';
+
+const EmailManagementHomePage = ( { source, selectedDomainName } ) => {
```

Comment:
Nit: it may be more appropriate to call this file `home-page` instead of just `page`, as we are currently losing the "Home" part of the component name.